components/public-api/gitpod/experimental/v1/user.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package gitpod.experimental.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/public-api/experimental/v1";
+
+import "google/protobuf/timestamp.proto";
+
+message User {
+    // id is a UUID of the user
+    string id = 1;
+
+    string name = 2;
+
+    string avatar_url = 3;
+
+    // created_at is the creation time
+    google.protobuf.Timestamp created_at = 5;
+}
+
+message PublicSSHKey {
+    // id is a UUID of the SSH key
+    string id = 1;
+
+    // name is the name of the SSH key
+    string name = 2;
+
+    // key is the public SSH key
```

Comment:
Is string the right representation? Or would we encode it as byte[]?

Reply:
I think string is fine, also the case in github api